maquetado/scss/titles.scss

```diff
@@ -0,0 +1,17 @@
+
+.title-text {
+  align-items: center;
+  display: flex;
+  font-size: $text-big;
+
+  &::before {
+    background-image: url("assets/triangle.svg");
+    background-repeat: no-repeat;
+    content: '';
+    height: $title-decoration-height;
```

Comment:
use only one of width and height to style image. this is to not change the aspect ratio of the image